docs/pages/web/table.js

```diff
@@ -2459,6 +2460,19 @@ function Example() {
 `}
           />
         </MainSection.Subsection>
+        <MainSection.Subsection
+          title="Selected & hovered state"
+          description={`Table.Row, Table.RowExpandable and Table.RowDrawer support hovered and selected states.
+
+If a row subcomponent is selectable, toggle the \`selected\` prop between "selected" and "unselected" to keep a constant border space in the row that is only visible when the row is selected.
+
+The \`selected\` prop should not be set, or set to "undefined", only if the row is not selectable. In this case, it does not set a border.
```

Comment:
grammar nit: the first sentence here is a little awkward. Consider switching the clauses, so starting with "If the row is not selectable…" and then the rest.

Reply:
Also, this recommendation doesn't match the Flow type. If we do want devs to be able to pass `undefined`, we need to add that to the enum of acceptable values.

Reply:
DONE